test/e2e/data/kubetest/conformance-networkpoicy.yaml

```diff
@@ -0,0 +1,8 @@
+ ## TODO(ashish-amarnath): get the correct focus string to run network policy tests
+ginkgo.focus: \[Conformance\]
```

Comment:
we should have a CI job for conformance and a separate one for the network policy as well.

Reply:
Yes, I agree. I didn't know the focus expression for those tests. so just leaving it like this for now till I find out what the right focus expression is.

Reply:
@jayunit100 will know for sure!